internal/measurex/measurer.go

```diff
@@ -413,19 +413,19 @@ func (mx *Measurer) quicHandshakeTimeout() time.Duration {
 
 // QUICHandshakeWithDB is like QUICHandshake but uses the given
 // db to store events rather than creating a temporary one and
-// use it to generate a new Measuremet.
+// use it to generate a new Measurement.
```

Comment:
Thanks! This is indeed following https://www.oreilly.com/library/view/97-things-every/9780596809515/ch08.html, which is great!